packages/block/test/difficulty.spec.ts

```diff
@@ -12,7 +12,6 @@ function runDifficultyTests(
 ) {
   const dif = block.canonicalDifficulty(parentBlock)
```

Comment:
Argh, I am not allow to comment on the `canonicalDifficulty()` method itself. 😜

So here: I would suggest that we rename this to `ethashCanonicalDifficulty()` to align with the clique-specific naming and also place this method further down the line directly above the `clique*()` methods to group these consensus-specific functions together.